tests/e2e/chain/config.go

```diff
@@ -164,10 +164,10 @@ func initGenesis(c *Chain) error {
 	}
 
 	// generate genesis txs
-	genTxs := make([]json.RawMessage, len(c.Validators))
-	for i, val := range c.Validators {
+	genTxs := make([]json.RawMessage, len(c.validators))
+	for i, val := range c.validators {
```

Comment:
Not relevant to this PR, but I think we should extract this into a separate function, and add some more docs on whats going on with stakeAmountCoin, its non-obvious to me

Reply:
Sure, I believe this is just adding the initial self-bond amount to each validator IIRC.